test/extended/templates/templateservicebroker_security.go

```diff
@@ -100,32 +109,36 @@ var _ = g.Describe("[templates] templateservicebroker security test", func() {
 
 	provision := func(username string) error {
 		g.By("provisioning a service")
-		_, err := brokercli.Provision(context.Background(), instanceID, &api.ProvisionRequest{
+		_, err := brokercli.Provision(context.Background(), &user.DefaultInfo{Name: username}, instanceID, &api.ProvisionRequest{
```

Comment:
I wonder whether you'll need to include the group "system:authenticated" on all of these - will add a follow-up question